lib/widgets/pages/home_page.dart

```diff
@@ -18,73 +48,53 @@ class HomePage extends StatefulWidget {
 
 class _HomePageState extends State<HomePage> {
   int _currentPageIndex = 0;
-
-  final List<Page> _pages = [
-    Page(
-      Strings.ticketsPageTitle,
-      Strings.ticketsNavTitle,
-      Icons.style,
-      TicketsPage(),
-    ),
-    Page(
-      Strings.receiptsPageTitle,
-      Strings.receiptsNavTitle,
-      Icons.receipt,
-      ReceiptsPage(),
-    ),
-    Page(
-      Strings.statsPageTitle,
-      Strings.statsNavTitle,
-      Icons.trending_up,
-      StatsPage(),
-    ),
-    Page(
-      Strings.settingsPageTitle,
-      Strings.settingsNavTitle,
-      Icons.settings,
-      SettingsPage(),
-    ),
-  ];
-
   Page get _currentPage => _pages[_currentPageIndex];
 
   void _onBottomNavTapped(int index) {
     setState(() => _currentPageIndex = index);
   }
 
   void _logout(BuildContext context) {
-    return BlocProvider.of<AuthBloc>(context).add(Unauthenticated());
+    return context.read<AuthBloc>().add(Unauthenticated());
   }
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: AppColor.background,
-      appBar: AppBar(
-        title: Text(_currentPage.appBarTitle, style: AppTextStyle.pageTitle),
-        actions: [
-          IconButton(
-            onPressed: () => _logout(context),
-            icon: const Icon(Icons.logout),
-          )
-        ],
-      ),
-      body: _currentPage.body,
-      bottomNavigationBar: BottomNavigationBar(
-        currentIndex: _currentPageIndex,
-        onTap: _onBottomNavTapped,
-        type: BottomNavigationBarType.fixed,
-        backgroundColor: AppColor.primary,
-        selectedItemColor: AppColor.white,
-        unselectedItemColor: AppColor.white.withOpacity(0.5),
-        selectedFontSize: 12,
-        unselectedLabelStyle: AppTextStyle.medium,
-        selectedLabelStyle: AppTextStyle.medium,
-        items: List.generate(
-          _pages.length,
-          (index) => BottomNavigationBarItem(
-            icon: Icon(_pages[index].icon),
-            label: _pages[index].navBarTitle,
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider(
+          create: (_) =>
```

Comment:
Any particular reason the creation of the ReceiptCubit was moved to the home page instead of the receiptsPage?

Reply:
Right, that makes sense. I forgot the same bloc would not be able to be reused, unless placed higher in the widget tree.